tdclient/table_api.py

```diff
@@ -47,15 +73,20 @@ def list_tables(self, db):
             return result
 
     def create_log_table(self, db, table):
-        """
-        TODO: add docstring
-        => True
+        """Create a new table in the database and registers it in PlazmaDB.
+
+        Args:
+            db (str): Target database name.
+            table (str): Target table name.
+
+        Returns:
+            bool: `True` if succeeded.
         """
         return self._create_table(db, table, "log")
 
     def create_item_table(self, db, table, primary_key, primary_key_type):
-        """
-        TODO: add docstring
+        """[Deprecated] Create a new table for item.
```

Comment:
Item tables had have already been deprecated, and, is not functioning anymore. Getting rid of the entire code block could make much more sense 🤔

Reply:
ok, will continue on another PR.